integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLivenessProbeCustomization.java

```diff
@@ -0,0 +1,487 @@
+                    .value("-Dweblogic.StdoutDebugEnabled=false"))
+                .addEnvItem(new V1EnvVar()
+                    .name("LIVENESS_PROBE_CUSTOM_SCRIPT")
+                    .value("/u01/appcheck.sh"))
+                .resources(new V1ResourceRequirements()
+                    .limits(new HashMap<>())
+                    .requests(new HashMap<>()))
+                .podSecurityContext(new V1PodSecurityContext()
+                    .runAsUser(0L)))
+            .adminServer(new AdminServer()
+                .serverStartState("RUNNING"))
+            .clusters(clusterList)
+            .configuration(new Configuration()
+                .model(new Model()
+                    .domainType(WLS_DOMAIN_TYPE)
+                    .runtimeEncryptionSecret(encryptionSecretName))));
+    setPodAntiAffinity(domain);
+    // create model in image domain
+    logger.info("Creating model in image domain {0} in namespace {1} using docker image {2}",
+        domainUid, domainNamespace, miiImage);
+    createDomainAndVerify(domain, domainNamespace);
+
+    // verify the admin server service and pod is created
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
+
+    // check the readiness for the managed servers in each cluster
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS_MIIDOMAIN; i++) {
+      for (int j = 1; j <= replicaCount; j++) {
+        String managedServerPodName =
+            domainUid + "-" + CLUSTER_NAME_PREFIX + i + "-" + MANAGED_SERVER_NAME_BASE + j;
+
+        // check managed server pod is ready and service exists in the namespace
+        logger.info("Checking that managed server pod {0} is ready and service exists in namespace {1}",
+            managedServerPodName, domainNamespace);
+        checkPodReadyAndServiceExists(managedServerPodName, domainUid, domainNamespace);
+      }
+    }
+
+    //check and wait for the application to be accessible in all server pods
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS_MIIDOMAIN; i++) {
+      for (int j = 1; j <= replicaCount; j++) {
+        String managedServerPodName =
+            domainUid + "-" + CLUSTER_NAME_PREFIX + i + "-" + MANAGED_SERVER_NAME_BASE + j;
+        String expectedStr = "Hello World, you have reached server "
+            + CLUSTER_NAME_PREFIX + i + "-" + MANAGED_SERVER_NAME_BASE + j;;
+
+
+        logger.info("Checking that application is running on managed server pod {0}  in namespace {1} with "
+             + "expectedString {3}", managedServerPodName, domainNamespace, expectedStr);
+        checkAppIsRunning(
+            domainNamespace,
+            managedServerPodName,
+            "8001",
+            "sample-war/index.jsp",
+            expectedStr);
+      }
+    }
+
+    logger.info("Domain {0} is fully started - servers are running and application is available",
+        domainUid);
+  }
+
+  private static void checkAppIsRunning(
+      String namespace,
+      String podName,
+      String internalPort,
```

Comment:
internalPort value is always "8001" and can be converted to a constant. Similarly, appPath value is always "sample-war/index.jsp" and can be converted to constant.

Reply:
Done

Reply:
Since it's a constant, it can be removed from the parameter list thus reducing the number of parameters. For e.g. `checkAppIsRunning` can be called like this.

        checkAppIsRunning(
            domainNamespace,
            managedServerPodName,
            expectedStr);

Reply:
Done